_posts/2018-10-25-released-v5.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Released Chainer/CuPy v5.0.0
+layout: post
+categories: Announcement
+author: Seiya Tokui
+---
+
+We have released Chainer and CuPy v5.0.0 today!
+This is a major release that introduces several new features.
+
+The following is a list of selected updates. Full updates can be found in the release notes: [Chainer](https://github.com/chainer/chainer/releases/tags/v5.0.0), [CuPy](https://github.com/cupy/cupy/releases/tag/v5.0.0).
+
+- **Static subgraph optimization** *(experimental)*.
+  By applying `@static_graph` decorator to the static part of your computation (which uses the same graph at every iteration), the computational graph of that part is cached and reused.
+  For fully-static models, we are observing 20-60% speed up in most cases.
+  Example code modified for the static subgraph feature can be found [here](https://github.com/chainer/chainer/tree/v5/examples/static_graph_optimizations).
+- **Float16 support**.
+  Using half-precision float is made much easier!
+  Since recent GPU technologies are focusing on half and mixed precision computations, using float16 is crucial for fully utilizing the latest hardware performance.
+  In Chainer v5, the default floating point dtype is configurable via `CHAINER_DTYPE` environment variable or `config.dtype` entry.
+  Using this feature, you can start trying float16 without modifying your code in most cases.
+  Many classes and functions are also fixed to support float16 inputs and parameters.
+- **ChainerMN integration**.
+  [ChainerMN](https://github.com/chainer/chainermn) is an add-on package of Chainer for distributed deep learning.
+  This is no longer an add-on package, but is a part of the official Chainer package from v5.
+  The APIs and the usage are not changed; you only need to install `chainer` and `mpi4py` to start distributed deep learning.
+- **Probability distributions**.
+  We introduced `chainer.distributions` module that implements many parametric probability distributions with autograd capability.
+  Each distribution provides point-wise evaluation (e.g. log density), statistics computation, and sampling.
+  For its implementation, we also added many GPU sampling routines (under `cupy.random`) and special functions (e.g. log-gamma function).
+  While v5 includes many frequently used distributions, we are still expanding this feature for the upcoming releases.
+- **iDeep 2.0**.
+  Chainer Backend for Intel Architecture, a.k.a. iDeep, is updated.
+  You can install it by `pip install ideep4py`, and use it by setting the environment variable `CHAINER_USE_IDEEP=auto`.
+  There are many performance improvements in this version.
+- **CuPy interoperability with other libraries and ecosystems**.
+  CuPy ndarray is now easily combined with other libraries.
```

Comment:
Let's link to the newly introduced Interoperability docs.
https://docs-cupy.chainer.org/en/latest/reference/interoperability.html

Reply:
added